sirepo/template/ml.py

```diff
@@ -326,6 +396,20 @@ def _get_matrix():
     )
 
 
+def _cols_with_non_unique_values(filename, has_header_row, header):
+    # TODO(e-carlin): support npy
+    assert not re.search(
+        r"\.npy$", str(filename)
+    ), f"numpy files are not supported path={filename}"
+    v = sirepo.numpy.ndarray_from_csv(_filepath(filename), has_header_row)
+    res = PKDict()
+    for i, c in enumerate(np.all(v == v[0, :], axis=0)):
+        if not c:
```

Comment:
unnecessary use of continue. Flip the condition.

Reply:
[bd53c46](https://github.com/radiasoft/sirepo/pull/4671/commits/bd53c468e02d539b2eb871f87914ee0c2a6c2f98)